bitvm/src/chunk/wrap_hasher.rs

```diff
@@ -41,113 +42,148 @@ pub(crate) mod hash_utils {
     /// Compute hash of top two field elements on stack: [a00, a01]
     /// Output is {BLAKE3_HASH_LENGTH} byte output represented in limb-form
     pub(crate) fn hash_fp2() -> Script {
-        script! {
-            // [a00, a01]
-            { hash_n_bytes::<64>() }
-            // Hash(a00|a01)
-            { pack_nibbles_to_limbs() }
-        }
+        static CACHE: OnceLock<Script> = OnceLock::new();
+        CACHE
+            .get_or_init(|| {
+                script! {
+                    // [a00, a01]
+                    { hash_n_bytes::<64>() }
+                    // Hash(a00|a01)
+                    { pack_nibbles_to_limbs() }
+                }
+            })
+            .clone()
```

Comment:
Why do we need to `clone` inside the cache?

Reply:
Beause `OnceLock::get_or_init` is returning `&T`

Reply:
Why not `&'static T`?